### DISCUSSION TITLE
Scoreboard

### DISCUSSION THREAD

**Discussion by itsashau** [Zrips/CMI#2883] [Feature Requests]
Adding a fully custom scoreboard for the plugin, allowing server owners to have access to turn it on and off if needed like the bossbar.
Access to placeholders from the PlaceholderAPI.

This would help out a lot.
Thanks

**Comment by Gyztor** [comment]
honestly would try out TAB (https://www.mc-market.org/resources/20631/) should work since it also has an entire customizable scoreboard that can even go on the bungeecord. (as the plugin can be installed on bungeecord and fully work) it might work out relatively well for you. though having some of the functionalities of this plugin included in CMI would also be really cool too tbh

**Comment by MCTGE** [comment]
Yes please, I would love to remove one more plugin from my server.

**Comment by kylemarham** [comment]
+1

**Comment by ChiefWarPig** [comment]
Would be great could ditch TAB as I only use it for Score Boards and TAB menu be another plugin CMI could let me cut from the server list.

**Comment by Angelius782** [comment]
yes it would be wonderful ... since simplescore is over and not maintain....... :(

**Comment by NightEye2004** [comment]
+1
